formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -157,21 +157,24 @@ let acceptsClosedAnonObjAsArg
     (o: < x : int, y : int >) =>
   o#x + o#y;
 
-let res = acceptsOpenAnonObjAsArg {
-  method x = 0;
-  method y = 10
-};
+let res =
+  acceptsOpenAnonObjAsArg {
+    method x = 0;
+    method y = 10
+  };
 
-let res = acceptsOpenAnonObjAsArg {
```

Comment:
True, this docks a brace at the end of a line, but it also docks `acceptsOpenAnonObjAsArg`, which is what I would rather it didn't do.  Instead I'd prefer a line break before `acceptsOpenAnonObjAsArg`, and then dock the brace at the end of that line:
```
let res =
  acceptsOpenAnonObjAsArg {
```